kedro_neptune/__init__.py

```diff
@@ -210,13 +207,13 @@ def _describe(self) -> Dict[str, Any]:
     def _save(self, data: bytes) -> None:
         load_path = get_filepath_str(self._get_load_path(), self._protocol)
```

Comment:
to further avoid confusion, you can change `load_path` -> `save_path` :)

Reply:
I've renamed it to just `path`.